content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -16,35 +16,46 @@ apply changes to a cluster.
 
 You can analyze your current live Kubernetes cluster by running:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=analyze_all_namespaces >}}
 $ istioctl analyze --all-namespaces
 {{< /text >}}
 
 And that’s it! It’ll give you any recommendations that apply.
 
 For example, if you forgot to enable Istio injection (a very common issue), you would get the following warning:
 
-{{< text plain >}}
-Warn [IST0102](Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< text syntax=plain snip_id=analyze_all_namespace_sample_response >}}
+...
+Warn [IST0102] (Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
 {{< /text >}}
 
+Fix the issue and try again:
+
+{{< text syntax=bash snip_id=fix_default_namespace >}}
+$ kubectl label namespace default istio-injection=enabled --overwrite
+$ istioctl analyze --namespace default
+✔ No validation issues found when analyzing namespace: default.
+{{< /text >}}
```

Comment:
The --overwrite is an example of where somebody reading the doc would say "why do I need to do that?". It doesn't matter if it fails or not if running it on the command line, it's only a problem for the test. So, rather than complicating the doc to accommodate the test, it's better to split this into two text blocks.
```suggestion
Fix the issue:

{{< text syntax=bash snip_id=fix_default_namespace >}}
$ kubectl label namespace default istio-injection=enabled --overwrite
{{< /text >}}

Then try again:

{{< text syntax=bash snip_id=try_with_fixed_namespace >}}
$ istioctl analyze --namespace default
✔ No validation issues found when analyzing namespace: default.
{{< /text >}}
```